atomistics/referencedata/wikipedia.py

```diff
@@ -3,14 +3,24 @@
 
 def get_elastic_properties(chemical_symbol: str) -> dict:
     """
-    Looks up elastic properties for a given chemical symbol from the Wikipedia: https://en.wikipedia.org/wiki/Elastic_properties_of_the_elements_(data_page) sourced from webelements.com.
+    Looks up elastic properties for a given chemical symbol from the Wikipedia: https://en.wikipedia.org/wiki/Elastic_properties_of_the_elements_(data_page)
+    sourced from webelements.com.
 
     Args:
         chemical_symbol (str): Chemical symbol of the element.
-        property (str): Name of the property to retrieve. Options: youngs_modulus, poissons_ratio, bulk_modulus, shear_modulus
 
     Returns:
-        str: Property value (various types): Value of the property for the given element, if available.
+        dict: Dictionary with the following keys
+            youngs_modulus: Young's modulus (or Young modulus) in GPa is a mechanical property of solid materials that
+                            measures the tensile or compressive stiffness when the force is applied lengthwise.
+            poissons_ratio: In materials science and solid mechanics, Poisson's ratio ν (nu) is a measure of the Poisson
+                            effect, the deformation (expansion or contraction) of a material in directions perpendicular
+                            to the specific direction of loading.
```

Comment:
```suggestion
            poissons_ratio: Ratio of transverse contraction strain to longitudinal extension strain in the direction of the stretching force
```

Reply:
but to be honest I think these explanations are not really necessary...

Reply:
The intention is to include the description of each parameter so that a LLM can leverage them. So while a human knows the reference for the LLM the additional context might be helpful.

Reply:
ok but I still think

```
poissons_ratio: Poisson's ratio (in GPa)
```

would be enough because LLM should know what Poisson's ratio is

Reply:
Maybe it helps less powerful LLMs like Llama. I would like to try it with more explicit Docstrings first and then we can optimise it later on.

Reply:
ok fine go ahead